legal-api/src/legal_api/models/filing.py

```diff
@@ -742,7 +742,8 @@ def set_processed(self, business_type):
     def effective_date_can_be_before_payment_completion_date(self, business_type):
         """For AR or COD filings on CP or BEN then the effective date can be before the payment date."""
         return self.filing_type in (Filing.FILINGS['annualReport'].get('name'),
-                                    Filing.FILINGS['changeOfDirectors'].get('name')) and business_type in {'CP', 'BEN'}
+                                    Filing.FILINGS['changeOfDirectors'].get('name')) and business_type in {'BEN','BC',
+                                     'C', 'CBEN','CP', 'CUL', 'ULC', 'CC', 'CCC'}
```

Comment:
Since its applicable for all legal type which can file a AR or COD
```suggestion
                                    Filing.FILINGS['changeOfDirectors'].get('name'))
```

Reply:
Thanks for clarifying, updated